pkg/util/log/strip.go

```diff
@@ -84,6 +84,13 @@ func matchCert() *regexp.Regexp {
 	)
 }
 
+// SetStrippedKeys allows configuration keys cleaned up
+func SetStrippedKeys(strippedKeys []string) {
+	configReplacer.Regex = matchYAMLKey(fmt.Sprintf("(%s)", strings.Join(strippedKeys, "|")))
+	configReplacer.Hints = strippedKeys
+	singleLineReplacers[5] = configReplacer
```

Comment:
Instead of replacing an index of this slice, which is not error-prone if somebody's editing the slice order or content without seeing this detail, we should consider not having this entry in the slice in the first place and then appending the replacer to it in this method?

Reply:
We could. I thought it would be more efficient, and I banged my head against it quite a bit.

Reply:
I agree that using an index is asking for trouble down the line.

I think we should not edit a default replacer but add a new one. This would also avoid duplicating the `(community_string|authKey|privKey)` default from line 55 in the configuration.

The default for the configuration would become an empty list and this function would just ignore empty list.

Reply:
Handled I think.